### ISSUE TITLE
Error: duplicate test title

### ISSUE THREAD

**Issue by YenovSegal** [microsoft/playwright#32462]
![image](https://github.com/user-attachments/assets/cf55478e-22ae-40b1-8b01-1010b078bdc5)

Error: duplicate test title "Multiple users › Create exam from the user › Set the user name to undefined", first declared in e2e\multiusers.spec.ts:311

**Comment by pavelfeldman** [comment]
userNameTwo and userNameThree are undefined

**Comment by reliablecontrols** [comment]
@YenovSegal  and @pavelfeldman . 
I experienced the same problem today (I have never experienced it before). 
I had a test that I copy and pasted. I got the appropriate error message  "duplicate test title"
then I renamed the test and resaved the file. Playwright in VS I have the same error message.

If you go to the testexplorer, and press refresh, the error clears itself up. 

version:
Last released
2024-10-22, 03:27:53
